package.json

```diff
@@ -78,14 +77,10 @@
   "main": "n/a",
   "scripts": {
     "develop": "gatsby develop",
-    "dev": "npm run develop",
     "serve": "gatsby serve",
     "build": "gatsby build",
     "build:pp": "gatsby build --prefix-paths",
-    "build:gh": "npm run clean && npm run build:pp && gh-pages -d public",
-    "clean": "rm -rf public && rm -rf .cache",
-    "deploy": "gatsby build && node ./scripts/deploy-github",
-    "render": "node ./scripts/deploy-render && gatsby build",
+    "clean": "gatsby clean",
```

Comment:
This seems to be a saner way to clean the outputs of Gatsby than `rm -f`-ing some directories.